xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1888,7 +1888,7 @@ bool CGUIWindowVideoBase::OnUnAssignContent(const std::string &path, int label1,
   bool bCanceled;
   CVideoDatabase db;
   db.Open();
-  if (CGUIDialogYesNo::ShowAndGetInput(label1, label2, label3, -1, bCanceled))
+  if (CGUIDialogYesNo::ShowAndGetInput(label1, label2, label3, "", bCanceled))
```

Comment:
@MartijnKaijser I'm not sure if this is an issue but `label1` is the header of the dialog in that case.